### PR TITLE
Added missing readme entry for the `actions_add_type_action` helper method.

### PR DESCRIPTION
**Pull request by phgross**
@jone please have a look ...

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -273,6 +273,12 @@ The ``UpgradeStep`` class has various helper functions:
     since all objects should be upgraded - even if the manager
     running the upgrades has no access on the objects.
 
+``self.actions_add_type_action(self, portal_type, after, action_id, **kwargs)``
+    Add a ``portal_type`` action from the type identified
```
Comment (jone) [inline README.rst]: ``portal_type`` -> ``portal_types`` (its the tool)

File: README.rst [hunk 2]
```diff
@@ -273,6 +273,12 @@ The ``UpgradeStep`` class has various helper functions:
...
+    by ``portal_type``, the position could be definded by the
+    ``after`` attribute. If the after action could not be found,
+    the action will be inserted at the end of the list.
```
Comment (jone) [inline README.rst]: I'd use the present tense, e.g. ``position CAN be DEFINED by``, ``after action CAN not be found``

Can you also update the method doc string?

**Comment by phgross** [comment]
@jone updated and rebased ...

**Comment by jone** [comment]
:+1:

**Pull request merged**